File: airport-web/src/app/shared/components/main-layout/shared/interfaces/flight.ts
import {Ticket} from "./ticket";
import {Airplane} from "../../../admin-layout/shared/components/airplane/shared/interfaces/airplane";

export interface Flight {
  id?: number,
  startPoint: string,
  endPoint: string,
  startDate: string,
  endDate: string,
  tickets?: Ticket[],
  airplaneId: number,
  airplane?: Airplane
}

File: airport-net/airport-net/Models/Flight.cs
namespace airport_net.Models
{
    public class Flight
    {
        public int Id { get; set; }
        public string StartPoint { get; set; }
        public string EndPoint { get; set; }
        public string StartDate { get; set; }
        public string EndDate { get; set; }
        public List<Ticket>? Tickets { get; set; }
        public int? AirplaneId { get; set; }
        public Airplane? Airplane { get; set; }
    }
}
